info/en_us/CONFIGURATION_OPTIONS.md

```diff
@@ -174,7 +174,7 @@ If the ISP/AS sent from the server is different from the one from Mojang Studios
 
 ### `resource-pack`
 
-URI to a resource pack
+URL to a resource pack
```

Comment:
No, URI is right

Reply:
oops, sorry. clicked the wrong thing there